src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -790,7 +785,7 @@ protected TokenTaskResult doInBackground(String... urlItems) {
                 // Record account in the AccountManager service
                 try {
                     setAccount(result);
-                } catch (Exception exc) {
+                } catch (GeneralSecurityException | IOException exc) {
```

Comment:
Well this is true if Logger.getInstance.setAndroidLogEnabled is not set to be false. Also, If externalLogger is used, we're losing the stacktrace.